components/ml/org.wso2.carbon.ml.commons/src/main/java/org/wso2/carbon/ml/commons/constants/MLConstants.java

```diff
@@ -177,6 +185,10 @@ private MLConstants() {
     public static final String MODEL_BUILDING_COMPLETE_NOTIFICATION = "modelBuildingCompleteNotification";
     public static final String MODEL_BUILDING_FAILED_NOTIFICATION = "modelBuildingFailedNotification";
 
+    //percentile range for anomaly detection
+    public static final int MIN_PERCENTILE = 80;
+    public static final int MAX_PERCENTILE = 100;
```

Comment:
How to change these percentile values (these should be the default ones)? Shall we try to read them from machine learner xml? May be as properties of algorithms?